yarpc/src/http/request.rs

```diff
@@ -49,7 +34,7 @@ impl Message for Request {
 
     fn first_line(&self) -> Vec<String> {
         [
-            self.method.to_owned(),
+            self.method.to_str().to_string(),
```

Comment:
```suggestion
            self.method.to_string(),
```
(This assumes Display being implemented on Method)